helpers/db.js

```diff
@@ -1,4 +1,5 @@
 import mongoose, { model, models, Schema } from "mongoose";
+import { v4 as uuidv4 } from "uuid";
 
 const URI = `mongodb+srv://Krokodil:${process.env.MONGODB_PASSWORD}@kroko.sd7bt9q.mongodb.net/?retryWrites=true&w=majority`;
```

Comment:
```suggestion
const URI = `mongodb+srv://Krokodil:${process.env.MONGODB_PASSWORD}@kroko.sd7bt9q.mongodb.net/your-database-name?retryWrites=true&w=majority`;
```

currently all your data should run onto a "test" database, you can change that bei adding your database name to the uri